infrastructure/prod.tfvars

```diff
@@ -1,4 +1,4 @@
 vault_section = "prod"
 s2s_url = "https://prod-s2s-api.reform.hmcts.net:3511"
 scheduling_enabled = "false"
-local_cache = "Always"
+local_cache = "Off"
```

Comment:
I would advise make if default off and then turn if on per environment through <env>.tfvars file.A good candidate for sandbox to see if there is any difference in 502 errors.Most importantly are we seeing any 502's on this service as CMC has not started using this so how we are going to verify the fix?